db/db_impl/db_impl.h

```diff
@@ -433,6 +436,7 @@ class DBImpl : public DB {
   struct GetImplOptions {
     ColumnFamilyHandle* column_family = nullptr;
     PinnableSlice* value = nullptr;
+    std::string timestamp;
```

Comment:
Can `timestamp` be of the same type as `value`?

Reply:
Changed to use std::string*. string implementation usually has internal space (15 bytes on Windows) and should be as good as PinnableSlice for timestamp. It avoids a conversion.